lib/client_side_validations/core_ext/regexp.rb

```diff
@@ -1,18 +1,8 @@
 class Regexp
+  require 'js_regex'
```

Comment:
moving it at the top of `lib/client_side_validations/core_ext/regexp.rb` (outside the `class Regexp`)?